package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@popperjs/core": "^2.11.8",
```

Comment:
Can't remember if I had intentionally omitted this but I'm comfortable adding it either way I think.

Reply:
Basically it's a dependency of bootstrap, but since peer dependencies are no longer automatically pulled in you would just get warnings all the times from Yarn. By pulling it in the warnings go away.